xbanish.c

```diff
@@ -15,16 +15,17 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
+#define _POSIX_C_SOURCE 2
+
 #include <err.h>
-#include <signal.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include <unistd.h>
+#include <stdbool.h>
+#include <strings.h>
 
-#include <X11/X.h>
 #include <X11/Xlib.h>
-#include <X11/Intrinsic.h>
```

Comment:
Hmm, I remember getting a failed compilation report due to this. It includes the header but doesn't actually link against the library.
https://bugs.gentoo.org/806258#c3